Slim/Player/TranscodingHelper.pm

```diff
@@ -407,6 +419,12 @@ sub getConvertCommand2 {
 			player           => $player || 'undefined',
 			channels         => $track->channels() || 2,
 			outputChannels   => $clientprefs ? ($clientprefs->get('outputChannels') || 2) : 2,
+			# aif/wav oddity #2 (for '-' rule)
+			# aif - aif: any SB that does not support wav requires aif header stripping
+			# wav/aif - pcm: force header stripping
+			stripHeader      => $caps->{'H'} || ($command eq "-" &&
```

Comment:
Oh my...

Reply:
Did I do something wrong or funny there?

Reply:
Well... you could call it "perly" :-). That `stripHeader` statement is quite convoluted. But it's ok!

Reply:
ah - I was thinking maybe something about my FrEnglish and the use of strip :-) 

But about being Perly, I had to admit that (remember I was born, with C) I enjoy some of the compactness you can have and although I'm still at low level, I like to play with regex and a
ll. Saves me so much code and loops ... And as we discussed, good code is when there is nothing more to remove (but not at the expense of being unreadable of course). 

And I'll blame you anyway because when you mentored me on the YT plugin, you pushed me in that direction :-)